catowseriOS/catowser/Browser/View/TabletView.swift

```diff
@@ -62,58 +82,106 @@ struct TabletView<C: BrowserContentCoordinators>: View {
         // with the search suggestions view when necessary
         showSearchSuggestions = false
         searchQuery = ""
-        searchBarState = .blankSearch
-        // Store references to subview models in the main view
-        // to be able to subscribe for the publishers
-        self.model = model
-        browserContentModel = BrowserContentModel(model.jsPluginsBuilder)
-        toolbarModel = WebBrowserToolbarModel()
-        searchBarModel = SearchBarViewModel()
+        searchBarAction = .clearView
         self.mode = mode
+        showingMenu = false
+        tabsCount = 0
     }
     
     var body: some View {
         switch mode {
         case .compatible:
             uiKitWrapperView
         case .full:
-            Spacer()
+            fullySwiftUIView
```

Comment:
Now tablet layout has full SwiftUI mode, but there is some issue with regular size classes